### PR TITLE
Expose sentry web command

### PR DESCRIPTION
**Pull request by JMSwag**

### CODE REVIEW

File: sentry/templates/deployment-sentry-web.yaml [hunk 1]
```diff
@@ -53,7 +53,7 @@ spec:
       - name: {{ .Chart.Name }}-web
         image: "{{ .Values.images.sentry.repository }}:{{ .Values.images.sentry.tag }}"
         imagePullPolicy: {{ .Values.images.sentry.pullPolicy }}
-        command: ["sentry", "run", "web"]
+        command: {{ .Values.sentry.web.comamnd }}
```
Comment (Mokto) [inline sentry/templates/deployment-sentry-web.yaml]: Hey, there is a typo here

Comment (JMSwag) [reply sentry/templates/deployment-sentry-web.yaml]: Updated 😉

**Comment by Mokto** [comment]
Can you bump the chart to 3.1.0 ?
Thanks.

**Comment by JMSwag** [comment]
Done. NP!

**Comment by JMSwag** [comment]
This is not ready to be merged. I'll update once I resolve the deployment issue.

**Comment by Mokto** [comment]
I'm going this as this as this has been more than one month it's opened.

Feel free to open another pull request ;)

**Pull request closed without merging**